cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -81,6 +81,37 @@ func CheckLocalEtcdClusterStatus(client clientset.Interface, cfg *kubeadmapi.Clu
 	return nil
 }
 
+// RemoveStackedEtcdMemberFromCluster will remove local etcd member from etcd cluster,
+// when reset the control plane node.
+func RemoveStackedEtcdMemberFromCluster(client clientset.Interface, cfg *kubeadmapi.InitConfiguration) error {
+	// creates an etcd client that connects to all the local/stacked etcd members
+	klog.V(1).Info("creating etcd client that connects to etcd pods")
+	etcdClient, err := etcdutil.NewFromCluster(client, cfg.CertificatesDir)
+	if err != nil {
+		return err
+	}
+
+	// notifies the other members of the etcd cluster about the joining member
+	etcdPeerAddress := etcdutil.GetPeerURL(cfg)
+
+	klog.V(2).Infof("get the member id from peer: %s", etcdPeerAddress)
+	id, err := etcdClient.GetMemberID(etcdPeerAddress)
+	if err != nil {
+		return err
+	}
+	klog.V(2).Infof("the member id of etcd peer %s is %d", etcdPeerAddress, id)
+
+	klog.V(1).Infof("Removing etcd member: %s", etcdPeerAddress)
```

Comment:
Maybe we can add the MemberID for this log and remove the two `klog.V(2)`, but I'm not sure, Wdyt?

Reply:
I remove the above `klog.V(2)`.